pkg/virt-api/api.go

```diff
@@ -277,10 +282,10 @@ func (app *virtAPIApp) composeSubresources(ctx context.Context) {
 		To(func(request *restful.Request, response *restful.Response) {
 			response.WriteAsJson(subresourceAPIGroup())
 		}).
-		Operation("getAPIGroup").
+		Operation("getSubresourceAPIGroup").
```

Comment:
On line 270 you have SubResources instead of Subresources (R/r) ... I believe it should be consistent also on line 299. I don't mind what variant you pick.

Reply:
Thx will fix it